crates/torii/migrations/20240207171639_add_txn_type.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE
+    transactions
+ADD
+    COLUMN transaction_type TEXT NOT NULL;
```

Comment:
I think we need a default here for backwards compat

Reply:
You're right. Fixed